src/components/organisms/Splash.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import { SPLASHKEY } from "../../constants/LOCAL_KEY";
+import imgUrl from "../../assets/splash.jpg";
+
+const Splash = () => {
+  const [isSplash, setIsSplash] = useState<boolean>(
+    localStorage.getItem(SPLASHKEY) === "false" ? false : true
+  );
+
+  useEffect(() => {
+    if (isSplash) {
+      setTimeout(() => {
+        setIsSplash(false);
+        localStorage.setItem(SPLASHKEY, "false");
+      }, 3000);
+    }
+  });
+
+  return <>{isSplash && <Wrapper src={imgUrl}></Wrapper>}</>;
+};
+
+const Wrapper = styled.img`
+  position: absolute;
+  width: 100%;
+  height: 100%;
```

Comment:
```suggestion
  width: 20rem;
  height: 35rem;
  z-index:99;
```

요 부분을 이렇게 변경하고 각 페이지에서 호출하지 않고 App에서 호출하면 반복해서 호출하는 부분을 줄일 수 있을 것 같아요!!!
요런 디테일... 너무 좋아요👍🏻👍🏻👍🏻👍🏻👍🏻👍🏻

Reply:
안 그래도 모든 페이지에 넣어두는 것이 지저분해보인다고 생각했는데 제안해주신 방법으로 바꿔볼게요! 😄